docs/pages/developer-guide.md

```diff
@@ -1137,20 +1137,20 @@ Here, the file `server.crt` contains an end-entity certificate in the OpenSSL PE
 
 # Regions and Zones
 
-Mesos allows agents to expose fault domain information in the form of a region and zone.  A region is larger than a zone and should be thought of as containing zones.  For example a region could be a particular datacenter and the racks within that datacenter could be its zones.  When this information is provided by Mesos it is injected into each Task's environment.  For example:
+Mesos allows agents to expose fault domain information in the form of a region and zone. A region is larger than a zone and should be thought of as containing zones. For example, a region could be a particular datacenter and the racks within that datacenter could be its zones. When this information is provided by Mesos, it is injected into each task's environment. For example:
 
 ```
 REGION: us-west-2
 ZONE: us-west-2a
 ```
 
-Services may choose to use this information to enable rack awareness.  When doing so, they should use placement rules to ensure that their pods are appropriately placed withing regions and zones.  One may apply placement constraints against regions and zones by referencing `@region` and `@zone` keys.  For example:
+Services may choose to use this information to enable rack awareness. When doing so, they should use placement rules to ensure that their pods are appropriately placed withing regions and zones. One may apply placement constraints against regions and zones by referencing `@region` and `@zone` keys.  For example:
 
 ```
 @zone:GROUP_BY:2
 ```
 
-The placement rule above would apply the `GROUP_BY` operator to zones.
+The placement rule above would apply to the `GROUP_BY` operator to zones.
```

Comment:
@sascala The operator is applied to the zones key.  The operator is how you write rules.  The rule doesn't apply _to_ the operator.

Reply:
oh duh. thank you